src/AdoPat.Test/PatManagerTest.cs

```diff
@@ -176,6 +198,13 @@ public async Task GetPatAsync_RegeneratesPatWhenPatHasExpired()
 
             // Assert
             pat.Should().BeEquivalentTo(expectedPat);
+
+            this.logTarget.Logs.Count.Should().Be(5);
+            this.logTarget.Logs[0].Should().Be($"Checking for PAT in cache with key '{this.cacheKey}'");

```

Comment:
TL;DR - I don't think this is an issue, but if you disagree it's good to pause and think about it more.

The cache key need not be confidential because it consists only of information that is supplied on the command line when the program is run, namely `organization`, `displayName`, and `scopes`. The formula for creating the cache key is to take the SHA256 hash of the aforementioned pieces of information and concatenate them with a `-`.

https://github.com/AzureAD/microsoft-authentication-cli/blob/cc9319143a6b15d8523616d9d3f0f478ff18150e/src/AdoPat/PatOptions.cs#L37-L56

Even if it were important to safeguard that information it is my understanding that the [SHA256](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.sha256?view=net-8.0) hashing algorithm is cryptographically secure. The original use for hashing the cache key was not actually security, but rather to ensure that the characters were fixed length, valid JSON keys, and to prevent accidental overlap where two similar keys might conflict.

Reply:
I should also note that the cache itself is still encrypted and locked with the user's credentials.